src/main/java/net/imglib2/cache/util/FunctorLoader.java

```diff
@@ -0,0 +1,44 @@
+package net.imglib2.cache.util;
+
+import java.util.stream.IntStream;
+
+import net.imglib2.FinalInterval;
+import net.imglib2.Interval;
+import net.imglib2.cache.CacheLoader;
+import net.imglib2.img.cell.Cell;
+import net.imglib2.img.cell.CellGrid;
+
+public class FunctorLoader< A > implements CacheLoader< Long, Cell< A > >
+{
+
+	public interface Functor< T, R > {
```

Comment:
@ctrueden `CacheLoader` also `throws Exception`. This was copied from Guava's `CacheLoader` class. I figured that they would have a good reason to do it?